flink/v1.16/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -316,6 +316,11 @@ public Builder setSnapshotProperty(String property, String value) {
       return this;
     }
 
+    public Builder toBranch(String branch) {
```

Comment:
I am not superconfident in this, but `branch` feels more natural to me. We do not use `toTable`, why **to**Branch?

Reply:
I share the same take as @amogh-jahagirdar. but I am not strongly opinionated here.

Reply:
Thanks for the quick response +1

Reply:
Sure thanks for the review here, it's important to make sure we get the API correct!